lib/mariaex/protocol.ex

```diff
@@ -323,6 +323,19 @@ defmodule Mariaex.Protocol do
     end
   end
 
+  def sanitize_query(query, conn) when is_pid(conn) do
+    {:ok, pool_ref, _mod, state} = DBConnection.Holder.checkout(conn, [])
```

Comment:
This is a private API and shouldnt be used. Though we can delete this because of above comment.

Reply:
If we can solve this issue by altering `prepare_insert` (as you suggested), we probably do not need to use the private API. Hopefully we can make it work.

Reply:
Done.